apps/cowswap-frontend/src/modules/hooksStore/hooks/useInternalHookDapps.ts

```diff
@@ -0,0 +1,20 @@
+import { useMemo } from 'react'
+
+import { useWalletInfo } from '@cowprotocol/wallet'
+
+import { ALL_HOOK_DAPPS } from '../hookRegistry'
+import { HookDapp } from '../types/hooks'
+
+export function useInternalHookDapps(isPreHook: boolean): HookDapp[] {
+  const { chainId } = useWalletInfo()
+
+  return useMemo(() => {
+    return ALL_HOOK_DAPPS.filter((dapp) => {
+      if (dapp?.conditions?.supportedNetworks && !dapp.conditions.supportedNetworks.includes(chainId)) return false
+
+      if (isPreHook && dapp?.conditions?.position && dapp.conditions.position !== 'pre') return false
```

Comment:
What it this is a post hook (`isPreHook === false`) and the hook only allows it to be set as pre hook?

Reply:
Thank you!
Fixed that: https://github.com/cowprotocol/cowswap/pull/4920/commits/1cf305ffd80658aac0146dee2a152458e879a082